runtime/oti/j9vm.hdf

```diff
@@ -276,6 +276,18 @@ typedef UDATA (* lookupNativeAddressCallback)(struct J9VMThread *currentThread,
 		<struct>J9VMReleaseVMAccessEvent</struct>
 		<data type="struct J9VMThread*" name="currentThread" description="current thread" />
 	</event>
+	
+	<event>
+		<name>J9HOOK_VM_ACQUIRING_EXCLUSIVE_IN_NATIVE</name>
+		<description>
+				Triggered during acquire exclusive thread loop, when encountered a thread in native (which we did not count for exclusive).
+				This hook serves similar purpose as J9HOOK_VM_RELEASEVMACCESS, to ensure that required work for thread in native
```

Comment:
Is the hook name misspelled here?

Reply:
J9HOOK_VM_RELEASEVMACCESS one? I'm intetionally refering to eat to give a hint to whoever wants to register something to J9HOOK_VM_RELEASEVMACCESS should also consider registering similar logic to J9HOOK_VM_ACQUIRING_EXCLUSIVE_IN_NATIVE (if atomic-free JNI is used).

Reply:
Thought maybe there were some `_` missing, but no.